src/interface/client.js

```diff
@@ -34,9 +30,13 @@ export function request(options) {
 
             options.window = el.contentWindow;
 
-            if (!options.window) {
-                throw new Error('Expected options.window');
-            }
+            
+        } else if (options.window.tagName && options.window.tagName.toLowerCase() === 'iframe') {
+            options.window = options.window.contentWindow;
+        }
+
+        if (!options.window || !options.window.document) {
+            throw new Error('Expected options.window to be a window object.');
```

Comment:
We should say something like `'Expected options.window to be a window object, iframe, or iframe element id.'`